### DISCUSSION TITLE
horizontal and vertical scrollbars do not work as smoothly

### DISCUSSION THREAD

**Discussion by alexsept** [flet-dev/flet#4178] [Q&A] [answered]
### Question

I can click on the Scroll Thumb on the vertical and horizontal scrollbar but I can't move it. I have to click on the area on the scrollbar, but that doesn't work as smoothly and seems a bit buggy to me. Since the latest update (0.24.1), the scrollbar doesn't work as it should.
Does anyone have any ideas? Thanks a million in advance!

### Code sample

```python
scrollable_table = ft.Container(
            content=ft.Column(
                controls=[
                    ft.Row(
                        controls=[
                            ft.Container(
                                content=self.table,
                                width=1000
                            )
                        ],
                        expand=True,
                        scroll=ft.ScrollMode.ALWAYS,  #horizontal
                    )
                ],
                scroll=ft.ScrollMode.ALWAYS,  #vertical
            ),          
            height=380,  # 
            width=1000
        )
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
On which OS are you? can you share runnable code that repoduces the issue?

**Reply by SUYinFu** [reply]
> On which OS are you? can you share runnable code that repoduces the issue?

windows 11 :The scroll bar cannot be dragged to move it. When will the issue be fixed?

**Reply by ndonkoHenri** [reply, marked as answer]
Can you try the latest pre-release: `pip install flet --pre -U`
If you face any issues, please provide repro code and possibly a screen record.

**Reply by SUYinFu** [reply]
> Can you try the latest pre-release: `pip install flet --pre -U` If you face any issues, please provide repro code and possibly a screen record.

Thank you, it worked.
